.cargo/config_fast_builds.toml

```diff
@@ -129,8 +129,8 @@ rustflags = [
     # "-Clink-arg=-fuse-ld=/opt/homebrew/opt/llvm/bin/ld64.lld",
 
     # Nightly
-    # "-Zshare-generics=y",
-    # "-Zthreads=0",
+    "-Zshare-generics=y",
+    "-Zthreads=0",
```

Comment:
Looks like you edited the template file?

Do you have a .cargo/config.toml with the changes?